### ISSUE TITLE
Change "esc to cancel" for mobile users

### ISSUE THREAD

**Issue by frdmn** [RocketChat/Rocket.Chat.Cordova#116]
... since they dont have a <kbd>ESC</kbd> key
##

**Comment by frdmn** [comment]
![img_1916](https://cloud.githubusercontent.com/assets/944459/15865470/2a7cfa5a-2cdb-11e6-85b3-d0eeaa123c02.jpg)

**Comment by dukemnm** [comment]
Hello @frdmn can you just delete the message using the back button? I wonder if this only shows on IOS?

**Comment by geekgonecrazy** [comment]
Agreed this is an irritating bit of UI.  We need to make it more mobile friendly.
